docs/deprecated.md

```diff
@@ -13,14 +15,12 @@ keywords: "docker, documentation, about, technology, deprecate"
      will be rejected.
 -->
 
-# Deprecated Engine Features
```

Comment:
These always surprise me (i.e.; why it can't automatically / organically pick the first H1 on the page).

Well, I guess it is what it is 🤷‍♂️